src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react"
+import { Box, Col, Flex, Grid, Row, Sans } from "@artsy/palette"
+import { Media } from "v2/Utils/Responsive"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { ArtistSeriesHeader_artistSeries } from "v2/__generated__/ArtistSeriesHeader_artistSeries.graphql"
+
+interface ArtistSeriesHeaderProps {
+  artistSeries: ArtistSeriesHeader_artistSeries
+}
+
+const ArtistSeriesHeader: React.FC<ArtistSeriesHeaderProps> = props => {
+  return (
+    <>
+      <Media greaterThanOrEqual="sm">
+        <ArtistSeriesHeaderLarge {...props} />
+      </Media>
+      <Media lessThan="sm">
+        <ArtistSeriesHeaderSmall {...props} />
+      </Media>
+    </>
+  )
+}
+
+const ArtistSeriesHeaderLarge: React.FC<ArtistSeriesHeaderProps> = props => {
+  const {
+    artistSeries: { title, description },
+  } = props
+  return (
+    <Box>
+      <Grid>
+        <Row>
+          <Col sm={6}>
+            <Flex
+              height="100%"
+              flexDirection="column"
+              justifyContent="space-between"
+            >
+              <Sans size="8" element="h1" unstable_trackIn>
```

Comment:
what's this `untable_trackIn` prop?

Reply:
This sets the tracking on the font (https://indesignskills.com/tutorials/letter-spacing-tracking-typography/).